measure.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 set -o errexit
 
-url=https://monitoring.googleapis.com/v1/projects/zeebe-io/location/global/prometheus/api/v1/query
+url=http://localhost:8001/api/v1/namespaces/monitoring/services/monitoring-kube-prometheus-prometheus:http-web/proxy/api/v1/query
 token=$(gcloud auth print-access-token)
 
+# open the proxy in the background
+kubectl proxy &
```

Comment:
:x: To be safe, let's hardcode the 8001 port, in case the default ever changes. You can specify here via `kubectl proxy --port=8001`